csrc/mmdeploy/codebase/mmdet/instance_segmentation.cpp

```diff
@@ -165,6 +169,7 @@ class ResizeInstanceMask : public ResizeBBox {
   operation::Managed<operation::WarpAffine> warp_affine_;
   float mask_thr_binary_{.5f};
   bool is_rcnn_{true};
+  bool is_resize_mask_{true};
```

Comment:
```suggestion
  bool is_resize_mask_{false};
```

I think it's better to make it false as default for backward compatible.

Reply:
👌